tutorials/20_score_based_methods_new_features.ipynb

```diff
@@ -0,0 +1,710 @@
+    "\n",
+    "- estimate the scores of the *tall* posterior for all times $t$ based on the learnt scores of **individual** posteriors $p(\\theta|x_0^i)$ for all $i$. $4$ different methods are implemented for the moment.\n",
+    "\n",
+    "- reverse the diffusion process or perform Langevin dynamics using the estimated scores  \n",
+    "\n",
+    "Note that in this i.i.d. setting, only the `\"sde\"` option is implemented for the moment.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 86,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": []
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "\u001b[A\n",
+      "Drawing 100 posterior samples: 100%|██████████| 499/499 [00:13<00:00, 37.00it/s]\n",
+      "Drawing 100 posterior samples: 100%|██████████| 100/100 [00:13<00:00,  7.41it/s]\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Build the tall posterior, and sample using sde option\n",
+    "tall_posterior = inference.build_posterior(score_estimator=score_estimator, prior=prior, sample_with=\"sde\")\n",
```

Comment:
we can omit `score_estimator=score_estimator` because we are just re-using the internally saved one. I think that's cleaner here.

Reply:
maybe add a note that the posterior object is essentially the same as above, just that we need to make sure we use "sde" for sampling. 

then in the line below, please add a note that we just call `sample` as before, but because we pass a batch of x, it will assume those are i.i.d. and automatically switch to the iid setting.